toolflow/vivado/common/common_2019.2.tcl

```diff
@@ -26,5 +26,6 @@ dict set stdcomps   mig_core         vlnv   "xilinx.com:ip:mig_7series:4.2"
 dict set stdcomps   ultra_ps         vlnv   "xilinx.com:ip:zynq_ultra_ps_e:3.3"
 dict set stdcomps   xxv_ethernet     vlnv   "xilinx.com:ip:xxv_ethernet:3.1"
 dict set stdcomps   100g_ethernet    vlnv   "xilinx.com:ip:cmac_usplus:3.0"
+dict set stdcomps   aurora           vlnv   "xilinx.com:ip:aurora_64b66b:12.0"
```

Comment:
You always need to add this for all relevant Vivado versions. So in this case at least 2019.1 and 2019.2 for XUPVVH

Reply:
Thanks, fixed